src/engine/sparqlExpressions/UuidExpressions.h

```diff
@@ -0,0 +1,73 @@
+//  Copyright 2024, University of Freiburg,
+//                  Chair of Algorithms and Data Structures
+//  Author: Hannes Baumann <baumannh@informatik.uni-freiburg.de>
+
+// Test for UuidExpression can be found in RandomExpressionTest.cpp
+
+#pragma once
+
+#include "engine/sparqlExpressions/SparqlExpression.h"
+#include "util/ChunkedForLoop.h"
+#include "util/Random.h"
+
+namespace sparqlExpression {
+
+using LiteralOrIri = ad_utility::triple_component::LiteralOrIri;
+
+inline constexpr auto fromLiteral = [](std::string_view str) {
+  return LiteralOrIri{
+      ad_utility::triple_component::Literal::literalWithNormalizedContent(
+          asNormalizedStringViewUnsafe(str))};
+};
+
+inline constexpr auto fromIri = [](std::string_view str) {
+  return LiteralOrIri{
+      ad_utility::triple_component::Iri::fromStringRepresentation(
+          absl::StrCat("<urn:uuid:"sv, str, ">"sv))};
+};
+
+inline constexpr auto litUuidKey = [](int64_t randId) {
+  return absl::StrCat("STRUUID "sv, std::to_string(randId));
```

Comment:
no need for the `to_string`,  absl handles this internally (same for the next function).
```suggestion
  return absl::StrCat("STRUUID "sv, randId);
```